utils/docker/examples/.env

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021-2022 Intel Corporation
```

Comment:
Date?  - perhaps this file is a rename that git didn't identify correctly?

Reply:
- [ ] Fix copyright date